src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -9,16 +9,28 @@
     [TestFixture]
     public class MsmqQueueCreatorTests
     {
+
+        [TearDown]
+        public void TearDown()
+        {
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.receiver");
```

Comment:
>Not sure I like this approach, seems likely to either forget to add a line here when adding a new test or forget to change the queue name when refactoring a test?

I think @timbussmann suggestion of making it convention based will make this better. 

>Also if you stop in the middle of debugging the queue will still be there and potentially screw up your next test run? (we should clear both before and after?)

As @bording pointed out the test should be more focused on what it is testing and not so much on the setting up / deleting the queue.  A Startup method can ensure that the queue doesn't exist.